.prettierrc

```diff
@@ -1,15 +1,81 @@
 {

```

Comment:
What is the source of all of this customization in this file?

The standard config is available [here](https://developer.salesforce.com/docs/platform/sfvscode-extensions/guide/prettier.html). I see advantage in specifying `apexInsertFinalNewline : false`, but the rest of the stuff seems to be the project-specific configuration from another team's project.

Reply:
This is the file I use for my projects - it has more granular items from my experiences. For example, the space before the closing `/>` on the top line of metadata files now won't trigger a whitespace change if the metadata is downloaded from the org.